justfile

```diff
@@ -122,6 +122,7 @@ test *ARGS: devenv
         --cov=tests \
         --cov-report=html \
         --cov-report=term-missing:skip-covered \
+        --no-migrations \
```

Comment:
Would you consider adding this to the `[tool.pytest.ini_options]` section of pyproject.toml instead?  That way, it'll be picked up even if the user doesn't use `just`.

Reply:
Oh yes, that's a much better idea actually 👍